### DISCUSSION TITLE
Seam issue help

### DISCUSSION THREAD

**Discussion by selphb** [OrcaSlicer/OrcaSlicer#1854] [General]
Looking for some help with my seam issue in the picture. I have gone into OrcaSlicer and reduced the ( Seam Gap ) and still can't get rid of it or reduce it. Any recommendation's would be greatly appreciated. I am using the slicer on my Bambu Labs printer. Any help would be a great help. 
![IMG_20230819_110432_01](https://github.com/SoftFever/OrcaSlicer/assets/57848350/7db75fde-97b0-4acf-960d-bab2a782be24)

**Comment by shrimpPark** [comment]
seam gap 1%
